roles/content_view_promotion_rollback_publish/vars/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Generally, I'd expect this file to be dropped from the final version given its specificity.